weblab_cg/data/tests/chaste_reference_models/cellml/aslanidi_model_2009.cellml

```diff
@@ -0,0 +1,4092 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Do we really need four separate files? If we do it properly, unit tests should have caught all functionality in nice little modular blocks, and big tests like this won't add any value. At the very least we could just select 1 big cellml file and edit it until it contains all the features we want? Or if there are very different ways of using a cellml file, then an example of each, but this just looks like 4 very similar testcases to me

Reply:
Eventually maybe not but for now yes. I've split the test into a develop test with the complicated symbolic test to help me develop and a straight textual comparison of models that I've already tested in chaste.